### PR TITLE
[jsk_2016_01_baxter_apc] visualize object recognition results

### PR DESCRIPTION
**Pull request by knorth55**
Visualize object recognition result and probability and object's region in image

rect_label_to_image

### CODE REVIEW

**Comment by knorth55** [comment]
#1058

File: jsk_2016_01_baxter_apc/node_scripts/rect_label_to_image.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+class RectLabelToImage(ConnectionBasedTransport):
+    def __init__(self):
+        super(RectLabelToImage,self).__init__()
+        self.pub = self.advertise("~output", Image, queue_size=10)
+
+    def subscribe(self):
+        self.sub_rect = message_filters.Subscriber('~input', RectArray)
+        self.sub_img = message_filters.Subscriber('~input/image', Image)
+        self.sub_result = message_filters.Subscriber('~input/result', ClassificationResult)
```
Comment (wkentaro) [inline jsk_2016_01_baxter_apc/node_scripts/rect_label_to_image.py]: why not `~input/classification`?

Comment (knorth55) [inline jsk_2016_01_baxter_apc/node_scripts/rect_label_to_image.py]: ```result``` looks good to me because ```classification``` is name of processing

Comment (wkentaro) [inline jsk_2016_01_baxter_apc/node_scripts/rect_label_to_image.py]: ah, no, `~input/class`.

Comment (knorth55) [inline jsk_2016_01_baxter_apc/node_scripts/rect_label_to_image.py]: that's better

**Comment by wkentaro** [comment]
This is really Nice!
I think the input for drawing result can be ClusterPointIndices or LabelImage, as well as the RectArray.
like:
- https://github.com/jsk-ros-pkg/jsk_recognition/pull/1579
- http://jsk-docs.readthedocs.org/en/latest/jsk_recognition/doc/jsk_perception/nodes/label_image_decomposer.html
So IMO, we should create a node named `visualize_object_detection` node which can subscribe
- jsk_recognition_msgs/RectArray
- jsk_recognition_msgs/ClusterPointIndices
- sensor_msgs/Image

In summarize, the design is as below:

package: jsk_perception
node_name: draw_object_detection.py
subscriptions:
- `~input`: `sensor_msgs/Image`
- `~input/classes`: `jsk_recognition_msgs/ClassificationResult`
- `~input/reference`: rosparam `~reference_msg`
publications:
`~output`: `sensor_msgs/Image`
parameters:
`~reference_msg`: messaage class of `~input/reference`.

**Comment by wkentaro** [comment]
LGTM, I will merge this soon.
It would be a great feature, if you add a abstract drawing node for object detection to `jsk_perception` package. (if you have time.)

**Pull request merged**